workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/LicenseCard/index.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+
+import './index.scss';
+
+import ClayButton from '@clayui/button';
+
+import infoCircleFullIcon from '../../../../assets/icons/icon_info_circle_full.svg';
+
+type LicenseTiers = {
+	quantity?: number;
+	value?: number;
+};
+
+const LicenseSectorCard: React.FC<any> = ({
+	icon = <img alt="Info" src={infoCircleFullIcon} />,
+	licenseDescription,
+	licensetiers,
```

Comment:
Please, change this name.
```suggestion
	licenseTiers,
```